ep/build-velox/src/get_velox.sh

```diff
@@ -58,6 +58,7 @@ function process_setup_ubuntu {
   sed -i '/^sudo --preserve-env apt update && sudo apt install -y/a\  uuid-dev \\' scripts/setup-ubuntu.sh
   sed -i 's/^  liblzo2-dev.*/  liblzo2-dev \\/g' scripts/setup-ubuntu.sh
   sed -i '/libre2-dev/d' scripts/setup-ubuntu.sh
+  sed -i '/libgmock-dev/d' scripts/setup-ubuntu.sh # resolved by ep/build-velox/build/velox_ep/CMake/resolve_dependency_modules/gtest.cmake
```

Comment:
@zhejiangxiaomai 

`libgmock` installed by the script is 1.10. The one installed by `resolve_dependency_modules/gtest.cmake` is 1.13.

I am removing his one. Any comment?

Reply:
let use 1.13

Reply:
OK. So I suppose this change is reasonable?

Reply:
Yes.

Reply:
Got it. Thanks for helping.